tests/spdx/parser/tagvalue/test_package_parser.py

```diff
@@ -83,6 +83,26 @@ def test_parse_package():
     assert package.valid_until_date == datetime(2022, 1, 1, 12)
 
 
+def test_parse_package_with_no_assertion_as_source_info():
```

Comment:
Ideally, this test should include all fields that can sport text only, all filled with either NONE or NOASSERTION. Might be bit overkill, though.
However, at least a field filled with NONE should be present.

Reply:
Adding all cases would be overkill imho but I added a field that contains NONE and is parsed as text.